book/core-concepts/partitioning.md

```diff
@@ -2,130 +2,69 @@
 
 If all of the application state exists in one state object then only one state computation at a time can access that state object. In order to leverage concurrency, that state needs to be divided into multiple distinct state objects. Wallaroo can then automatically distribute these objects in a way that allows them to be accessed by state computations in parallel.
 
-For example, in an application that keeps track of stock prices, the application state might be a dictionary where the stock symbol is used to look up the price of the stock.
+For example, in an application that keeps track of stock prices, the naive application state might be a dictionary where the stock symbol is used to look up the price of the stock.
```

Comment:
Wicked nit-picky, but I so prefer naïve over naive. I actually looked it up and "naive" is valid, so up to you.

Reply:
done